flow/layers/shader_mask_layer_unittests.cc

```diff
@@ -373,17 +373,18 @@ TEST_F(ShaderMaskLayerTest, OpacityInheritance) {
     {
       expected_builder.translate(offset.fX, offset.fY);
       /* ShaderMaskLayer::Paint() */ {
-        expected_builder.setColor(opacity_alpha << 24);
-        expected_builder.saveLayer(&child_path.getBounds(), true);
+        DlPaint sl_paint;
+        sl_paint.setColor(opacity_alpha << 24);
+        expected_builder.saveLayer(&child_path.getBounds(), &sl_paint);
         {
           /* child layer paint */ {
-            expected_builder.setColor(0xFF000000);
-            expected_builder.drawPath(child_path);
+            expected_builder.drawPath(child_path,
+                                      DlPaint().setColor(0xFF000000));
```

Comment:
I think that's the default color, no? The reason we had a setColor() before was because the state from the saveLayer was still in the DL buffer. Just using "builder.drawPath(child_path, DlPaint())" should be enough for this call.